src/test/java/LoginTests.java

```diff
@@ -0,0 +1,54 @@
+
+import Pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTests extends BaseTests {
+    @Test
+    void loginSuccessTest() {
+        //1. Login Success.
+
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.openLoginPage();
+        loginPage.enterEmail("alekseimaslo@gmail.com");
+        loginPage.enterPassword("Kwidos123");
+        loginPage.clickSubmit();
+        WebElement verifiedUserName = driver.findElement(By.xpath("//*[contains (text(), 'Aleksei')]"));
+        Assert.assertTrue(verifiedUserName.isDisplayed());
+
+
+    }
+
+    @Test
+    void loginWrongCredentialsTest() {
+        //2. Login Wrong credentials (not existing email)
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.openLoginPage();
+        loginPage.enterEmail("WrongAlekseimaslo@gmail.com");
+        loginPage.enterPassword("Kwidos123");
+        loginPage.clickSubmit();
+        String alert = driver.findElement(By.xpath("//div[@class='alert alert-danger']")).getText();
```

Comment:
It would be better to create a method getErrorMessage in LoginPage class. Do not use WebDriver methods in Test layer